Modules/Core/Repositories/BaseRepository.php

```diff
@@ -98,4 +98,10 @@ public function getByAttributes(array $attributes, $orderBy = null, $sortOrder =
      * @return bool
      */
     public function clearCache();
+
+    /**
+     * @param string $field
+     * @param array $values
+     */
+    public function whereIn(string $field, array $values);
```

Comment:
You could add the return typehint to be `\Illuminate\Database\Eloquent\Builder`. And since we're using PHP 7.1+, we don't need the docblock, just use PHP's typehints. Don't change this unless @nWidart says OK, though.

Reply:
+1